components/sendgrid/sources/new-contact/README.md

```diff
@@ -0,0 +1,378 @@
+will then be filtered like this:
+
+```sql
+created_at BETWEEN
+  TIMESTAMP "2020-11-08T19:15:00" AND
+  TIMESTAMP "2020-11-08T20:00:00"
+```
+
+The calculation is the following:
+
+1. Current execution timestamp is `20:00:00`
+2. Previous execution timestamp is `19:45:00` (given that the timer is set at 15
+   minutes)
+3. 30 minutes prior to the previous execution is `19:15:00`
+4. Ergo, the overall time range for which to search new contacts goes from
+   `19:15:00` (as per step 3) and `20:00:00` (as per step 1)
+
+Every record created between **19:15** and **19:30 UTC** will very likely show
+up in those search results. Records created within the last 30 minutes (i.e.
+from **19:30** to **20:00 UTC**) might not appear in the search results queried
+by the current execution, and will need to be retrieved during upcoming ones
+(i.e. the 4th execution or later).
+
+### Results Limits <a name="results-limits"></a>
+
+The 2nd [limitation](#limitations) points out that the search API does not
+provide an exhaustive list of contacts in a single search query, which means
+that this event source is likely to have to issue multiple queries within a
+certain time range to make sure the search is exhaustive.
+
+This is accomplished by:
+
+- Using the `contact_count` attribute of the search results. This attribute's
+  value is the actual number of records that match the search query, and can be
+  arbitrarily big. When `contact_count` is lower than 50, it will of course
+  match the amount of records in the search result.
+- Keeping track of all the record ID's returned for the search query. We use
+  those records in subsequent search queries to exclude them from search
+  results. So each time we issue a search query for a time range, we append
+  those ID's to the list of already-processed records and persist such list so
+  that subsequent executions can use it.
+- Keeping the time range across subsequent executions (fairly) constant so that
+  we continue retrieving the records created during such range. The time range
+  can be safely adjusted as we retrieve and process records, by inspecting the
+  `created_at` attribute of the last processed record, and the amount of records
+  left to retrieve (e.g. if there are few enough records to retrieve, we can
+  extend the upper bound of the time range to try and retrieve more records next
+  time).
+
+Users have the possibility to provide the number of new contacts to process
+within a single execution through the `limit` prop. This parameter is useful in
+cases when the search results return a number of new contacts higher that
+`limit`. Please note that the parameter is **optional** and its default value is
+**50** (which is the original limit imposed by the SendGrid API).
+
+#### Example
+
+We'll go through an example to illustrate the approach detailed above. For
+simplicity, let's make the following assumptions:
+
+- The search API returns a maximum of 2 records per call, which is also the
```

Comment:
is this comment right? the default for `limit` prop is set to 50

Reply:
I'll rephrase it to make it clearer that it's an assumption for the example rather than the actual default value.
You're right though, the default value for the `limit` prop is 50.